camayoc/tests/qcs/api/v1/scanjobs/test_run_scanjobs.py

```diff
@@ -209,6 +214,8 @@ def run_all_scans(vcenter_client, module_cleanup):
                         'source_ids', []).append(
                         source_ids[hostname])
     for scan in scans:
+        # grab the disabled products if they exist, otherwise {}
+        disabled_optional_products = scan.get('disabled_optional_products', {})
```

Comment:
So this is a section you are adding in the config file?

Reply:
Yes, I felt like this made the most sense with how the tests for scanjob are structured. For any defined scan in the config file you can just add the disabled_optional_product section:
```scans:
        - name: Fuse
          sources:
              - 'sonar.trial'
          disabled_optional_products: {'jboss_fuse': True}```
I put an example in the docstring of this function (line 139) but if I need to change this/change the structure of how it is in the config file, I can!

Reply:
Ah Ok! The way the diff folded out some lines I did not realize that you had put that line in the config file. Sounds good to me.